### ISSUE TITLE
can't find files directory in version 1.39.5

### ISSUE THREAD

**Issue by schyen** [mothur/mothur#364]
Hi,

One of the features I really like in mothur is that it will search through input, output, default and mothur directories when it is trying to find a file. However, in 1.39.5, I found that when it can't find a file in the input directory, it is unable to find the file in the other directories because it tacks on the word 'mothur' to the end of the path supplied. I didn't experience this with version 1.37.2, however.

For example:
```
mothur > set.dir(input=/my/fastq/folder/, output=/my/output/folder/, tempdefault=/path/to/my/ref/)

Mothur's directories:
outputDir=/my/output/folder/
inputDir=/my/fastq/folder/
tempDefault=/path/to/my/ref/

mothur > trim.seqs(fasta=my.fasta, qfile=my.qual, qthreshold=20, inputdir=/my/fastq/folder/, outputdir=/my/output/folder/, processors=8)
Setting output directory to: /my/output/folder/
Setting input directory to: /my/fastq/folder/
Unable to open /my/fastq/folder/my.fasta. Trying default /path/to/my/ref/my.fasta
Unable to open /path/to/my/ref/my.fasta. Trying mothur's location /path/to/mothur/mothurmy.fasta
Unable to open  /path/to/mothur/mothurmy.fasta. Trying output directory /my/output/folder/mothurmy.fasta
Unable to open /my/output/folder/mothurmy.fasta
...

```

**Comment by mothur-westcott** [comment]
Thanks for reporting this.  It looks like mothur is missing the '/' between the filename and the directory.  This will be fixed in version 1.40.0.

**Comment by schyen** [comment]
Thank you!
